planning/planners/src/encode/symmetry.rs

```diff
@@ -206,32 +223,36 @@ fn add_plan_space_symmetry_breaking(pb: &FiniteProblem, model: &mut Model, encod
         //     }
         // }
 
-        for (i, instance) in instances.iter().copied().enumerate() {
-            let mut clause = Vec::with_capacity(64);
+        for (i, crt_instance) in instances.iter().copied().enumerate() {
```

Comment:
@Shi-Raida An extended comment on why this is sound (similar to what we did with a pen and paper would be helpful).